runner.go

```diff
@@ -43,6 +43,7 @@ type runner struct {
 
 	numClients int32
 	spawnRate  float64
+	targetHost string
```

Comment:
I don't want to keep this field. Publishing an event is enough.

Reply:
I removed the field in a553bd9737196a8d180455d773207a44b70421c3 .